### PR TITLE
Commodity input multiselect support

### PR DESCRIPTION
**Pull request by chrismou**
- Adds multi select support to commodity field
- Updates Commodity/Station forms 

Have pushed as a mini PR rather than bogging the change down in my Trade Route PR and risking conflicts later

### CODE REVIEW

File: app/_components/inputs/Commodities.tsx [hunk 1]
```diff
@@ -33,31 +40,30 @@ const CommoditiesField: React.FC<CommodityProps> = ({
     }));
   }
 
+  const fieldOptions: FieldOptions = {};
+  if (isMulti) {
+    fieldOptions.isMulti = true;
+  }
+
```
Comment (nitishdhar) [inline app/_components/inputs/Commodities.tsx:47]: Not a blocker: What other options do you anticipate for this? If not, you can just pass it in the Select component.

**Review by nitishdhar — APPROVED** [review]
Minor comment requesting clarity, otherwise lgtm

File: app/_components/inputs/Commodities.tsx (hunk 1 above)
Comment (chrismou) [reply app/_components/inputs/Commodities.tsx:47]: @nitishdhar You can't pass `isMulti={false}` on a react-select, it either accepts `true` or simply can't be there. So the simplest way of dealing with this, I figured, would be to populate an options object and spread it onto the Select, so if isMulti is set to false via the component prop it just wouldn't be included.

Am open to other ideas if you know of a more elegant way of dealing with it

image

Comment (nitishdhar) [reply app/_components/inputs/Commodities.tsx:47]: Nah, this is good then. Thanks for clarifying. Weird they don't take false there.

**Pull request merged**